testing_app/test/perf_test_e2e.dart

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This file duplicate the behavior of `test_driver/perf_test.dart` but uses
+// e2e package to implement a host-independent test.
+
+import 'package:e2e/e2e.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:testing_app/main.dart' as app;
+
+import 'e2e_utils.dart';
+
+void main() {
+  final binding =
+      E2EWidgetsFlutterBinding.ensureInitialized() as E2EWidgetsFlutterBinding;
+  // fullyLive frame policy simulates the way Flutter response to animations.
```

Comment:
"The fullyLive frame policy simulates the way Flutter responds to animations"?

Reply:
Done